src/test/java/org/jenkinsci/plugins/scriptler/ScriptlerManagementTest.java

```diff
@@ -0,0 +1,46 @@
+package org.jenkinsci.plugins.scriptler;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.markup.MarkupFormatter;
+import hudson.markup.RawHtmlMarkupFormatter;
+import java.io.IOException;
+import java.io.Writer;
+import org.junit.jupiter.api.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.junit.jupiter.WithJenkins;
+
+@WithJenkins
+class ScriptlerManagementTest {
+
+    @Test
+    void markupFormatter(@SuppressWarnings("unused") JenkinsRule r) throws IOException {
+        ScriptlerManagement management = new ScriptlerManagement();
+
+        // save text
+        String text = management.getMarkupFormatter().translate("Save text");
+        assertEquals("Save text", text);
+
+        // dangerous text with global formatter
+        text = management.getMarkupFormatter().translate("<script>alert('PWND!')</script>");
```

Comment:
This "PWND!" script is repeated 3 times. Can we extract it out to a static variable?